digdag-standards/build.gradle

```diff
@@ -28,4 +28,7 @@ dependencies {
     // https://bugs.eclipse.org/bugs/show_bug.cgi?id=475546
     // https://github.com/eclipse/jetty.project/issues/416
     compile 'org.eclipse.jetty:jetty-client:9.3.11.v20160721'
+
+    // XXX (dano): This is just here so we can disable TDClientConfing spam in TdDigdagClientConfigurationPlugin
+    compile 'ch.qos.logback:logback-classic:1.1.5'
```

Comment:
First, I thought that adding following lines to `digdag-cli/src/main/resources/digdag/cli/logback-color.xml` and `logback-console.xml` files is better. However, it violates separation of td-related modules in digdag-standards from digdag-cli. hm...I hope logback has better programable API...

```
  <!-- silence spam in TDClientConfig used at TdClientConfigProvider -->
  <logger name="com.treasuredata.client.TDClientConfig" level="WARN"/>
```